src/App.js

```diff
@@ -0,0 +1,58 @@
+import React from "react";
+import "./App.css";
```

Comment:
You are importing an unused css! Try always to import just the necessary files, so the user get the smallest app size